ads/google/a4a/utils.js

```diff
@@ -168,7 +168,6 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
-        {name: 'ea', value: '0'},
```

Comment:
I have not encountered it once in the wild while debugging this PR and others, both in AdSense and DoubleClick.